app/src/pages/inside/launchesPage/launchSuiteGrid/hamburger/hamburger.jsx

```diff
@@ -172,36 +171,24 @@ export class Hamburger extends Component {
               <FormattedMessage id={'Hamburger.export'} defaultMessage={'Export:'} />
             </div>
             <div className={cx('export-buttons')}>
-              <div className={cx('export-button')}>
-                <GhostButton
-                  tiny
-                  onClick={() => {
-                    onExportPDF(launch);
-                  }}
-                >
-                  PDF
-                </GhostButton>
-              </div>
-              <div className={cx('export-button')}>
-                <GhostButton
-                  tiny
-                  onClick={() => {
-                    onExportXLS(launch);
-                  }}
-                >
-                  XLS
-                </GhostButton>
-              </div>
-              <div className={cx('export-button')}>
-                <GhostButton
-                  tiny
-                  onClick={() => {
-                    onExportHTML(launch);
-                  }}
-                >
-                  HTML
-                </GhostButton>
-              </div>
+              <a
```

Comment:
Use GhostButtons on open required window.location in onClick callback or add style for your links in case when button pressed.